green_cli/common.py

```diff
@@ -231,15 +231,16 @@ def setpin(session, pin, device_id):
     return context.authenticator.setpin(session, pin, device_id)
 
 @green.command()
-@click.option('--password', type=str, default='')
+@click.option('--password', default="", expose_value=False, callback=details_json)
 @with_login
 @print_result
-def getmnemonic(session, password):
+@with_gdk_resolve
+def getmnemonic(session, details):
```

Comment:
This will now return credentials, not the mnemonic, so should be renamed.

Reply:
Renamed to `getcredentials`